tox.ini

```diff
@@ -13,7 +13,7 @@ markers =
 deps =
     .[test]
 commands =
-    flake8 cdptools --count --verbose --max-line-length=127 --show-source --statistics
+    flake8 cdptools --count --verbose --max-line-length=127 --show-source --statistics --ignore E501 cdptools/tests/event_scrapers
```

Comment:
Instead of adding this to the overall tox check file add this as a comment to each of the failing lines.